guides/common/modules/proc_adding-a-bonded-interface.adoc

```diff
@@ -50,9 +50,11 @@ endif::[]
 --managed true \
 --interface="identifier=_My_NIC_1, mac=_My_MAC_Address_1_, managed=true, type=Nic::Managed, domain_id=_My_Domain_ID_, subnet_id=_My_Subnet_ID_" \
 --interface="identifier=_My_NIC_2_, mac=_My_MAC_Address_2_, managed=true, type=Nic::Managed, domain_id=_My_Domain_ID_, subnet_id=_My_Subnet_ID_" \
---interface="identifier=_bond0_, ip=_My_IP_Address_2_, type=Nic::Bond, mode=active-backup, attached_devices=[_My_NIC_1_,_My_NIC_2_], managed=true, domain_id=_My_Domain_ID_, subnet_id=_My_Subnet_ID_" \
+--interface="identifier=_bondN_, ip=_My_IP_Address_2_, type=Nic::Bond, mode=active-backup, attached_devices=[_My_NIC_1_,_My_NIC_2_], managed=true, domain_id=_My_Domain_ID_, subnet_id=_My_Subnet_ID_" \
 --location "_My_Location_" \
 --name "_My_Host_Name_" \
 --organization "_My_Organization_" \
 --subnet-id=_My_Subnet_ID_
 ----
++
+Ensure to replace `bondN` with `bond` and the ID of your your device identifier, for example, `bond0`.
```

Comment:
```suggestion
Ensure to replace `bondN` with `bond` and the ID of  your device identifier, for example, `bond0`.
```

Reply:
Thanks for your ACK; applied your suggestion.